site-manage/site-manage-participant-helper/src/bundle/org/sakaiproject/site/tool/participant/bundle/sitesetupgeneric_sv.properties

```diff
@@ -477,16 +477,6 @@ man.spec=Speciella instruktioner\:
 man.subject=\u00c4mne\:(ie PSYCH)\t
 man.this=denna kurssajt\:
 man.toaddto=f\u00f6r att l\u00e4gga till p\u00e5
-multipleTools.custom=L\u00e4gg till flera verktyg
-multipleTools.custom1=L\u00e4gg till flera verktyg eller konfigurera verktygsinst\u00e4llningar. Du kan \u00e4ven g\u00f6ra \u00e4ndringar i verktygsinst\u00e4llningarna senda p\u00e5 sajten genom att anv\u00e4nda menyn Inst\u00e4llningar p\u00e5 respektive verktyg.
-multipleTools.email=E-postarkiv
-multipleTools.emailreq=E-postadress (obligatorisk)
-multipleTools.more=fler
-multipleTools.moreTools.1=Fler
-multipleTools.moreTools.2=verktyg?
-multipleTools.site=Sajtens e-postadress
-multipleTools.title=Namn
-multipleTools.title.suggest=(f\u00f6reslagen l\u00e4ngd \u00e4r 15 tecken)
```

Comment:
I'm presuming these are not used anymore?

Reply:
They belong to the file from the other component (site-manage-tool, not participant helper).